run_train.sh

```diff
@@ -13,6 +13,8 @@ set -ex
 NGPU=${NGPU:-"8"}
 export LOG_RANK=${LOG_RANK:-0}
 CONFIG_FILE=${CONFIG_FILE:-"./torchtitan/models/llama3/train_configs/debug_model.toml"}
+# This breaks compile tests
```

Comment:
nit - maybe clearer to say "turn off static launcher to avoid issues with CI GPU compile tests".
atm it reads like you are flagging that this current env var breaks compile tests.

Reply:
entirely fair. it also looks like this env var isn't working properly, so i'm probably just going to throw away this PR and disable the compile tests for now.